src/devtools/client/debugger/src/actions/sources/blackbox.ts

```diff
@@ -33,7 +33,7 @@ async function blackboxActors(
   return { isBlackBoxed: !isBlackBoxed };
 }
 
-export function toggleBlackBox(cx: Context, source: Source): UIThunkAction {
+export function toggleBlackBox(cx: Context, source: Source | SourceWithContent): UIThunkAction {
```

Comment:
Do you think this will stay typed as `Source | SourceWithContent`? Or is that a temporary type until we get to phase II?